scripts/pylint_extensions.py

```diff
@@ -1385,7 +1385,11 @@ def process_module(self, node):
 
 
 class SpaceBelowFileOverviewChecker(checkers.BaseChecker):
-    """Checks if there is an empty line between file overview and imports."""
+    """Checks if there is an empty line between file overview and imports.
```

Comment:
This docstring should not refer to imports. Perhaps something like "the next non-empty line", or you can just say "a single empty line below the fileoverview docstring".

Reply:
done